File: app/src/main/java/jogja/telkom/csrassistant/MainActivity.java
package jogja.telkom.csrassistant;

import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;
import jogja.telkom.csrassistant.AddOn.AddOnActivity;
import jogja.telkom.csrassistant.Isolir.IsolirActivity;
import jogja.telkom.csrassistant.Kendala.KendalaActivity;
import jogja.telkom.csrassistant.Tagihan.TagihanActivity;
import jogja.telkom.csrassistant.Upgrade.UpgradeActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        CardView add      = findViewById(R.id.add_on);
        CardView bill     = findViewById(R.id.bill);
        CardView upgrade  = findViewById(R.id.up_down);
        CardView unlock   = findViewById(R.id.unlock);
        CardView problem  = findViewById(R.id.problem);

        add.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, AddOnActivity.class));
        });


        bill.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, TagihanActivity.class));
        });


        upgrade.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, UpgradeActivity.class));
        });


        unlock.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, IsolirActivity.class));
        });


        problem.setOnClickListener(v -> {
            startActivity(new Intent(MainActivity.this, KendalaActivity.class));
        });

    }
}

File: app/src/main/java/jogja/telkom/csrassistant/MainActivity.kt
package jogja.telkom.csrassistant

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.cardview.widget.CardView
import jogja.telkom.csrassistant.AddOn.AddOnActivity
import jogja.telkom.csrassistant.Isolir.IsolirActivity
import jogja.telkom.csrassistant.Tagihan.TagihanActivity
import jogja.telkom.csrassistant.Upgrade.UpgradeActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val add = findViewById<CardView>(R.id.add_on)
        val bill = findViewById<CardView>(R.id.bill)
        val upgrade = findViewById<CardView>(R.id.up_down)
        val unlock = findViewById<CardView>(R.id.unlock)

        add.setOnClickListener { v -> startActivity(Intent(this@MainActivity, AddOnActivity::class.java)) }


        bill.setOnClickListener { v -> startActivity(Intent(this@MainActivity, TagihanActivity::class.java)) }


        upgrade.setOnClickListener { v -> startActivity(Intent(this@MainActivity, UpgradeActivity::class.java)) }


        unlock.setOnClickListener { v -> startActivity(Intent(this@MainActivity, IsolirActivity::class.java)) }

    }
}
